zerver/views/presence.py

```diff
@@ -54,13 +54,27 @@ def get_presence_backend(request: HttpRequest, user_profile: UserProfile,
 @has_request_variables
 def update_user_status_backend(request: HttpRequest,
                                user_profile: UserProfile,
-                               away: bool=REQ(validator=check_bool),
+                               away: Optional[bool]=REQ(validator=check_bool, default=None),
+                               info: Optional[str]=REQ(default=None),
                                ) -> HttpResponse:
 
+    if info is not None:
+        info = info.strip()
+
+        # We will start with a short limit for now till we
+        # get some UI feedback.  We can always increase this
+        # later.
+        if len(info) > 60:
+            return json_error(_('Please limit info to 60 characters.'))
```

Comment:
We should just use `check_capped_string(60)` -- avoids adding a new string for translators.